### PR TITLE
add logging aspect tests

### PR DESCRIPTION
**Pull request by daniel413x**

### CODE REVIEW

**Review by rey1776 — APPROVED** [review]

**Comment by brittshook** [comment]
I get this error running the tests:

```
[INFO] 
[ERROR] Failures: 
[ERROR]   TestLoggingAspect.testLogException:74 
Wanted but not invoked:
mockProceedingJoinPoint.proceed();
-> at com.skillstorm.taxservice.aspects.TestLoggingAspect.testLogException(TestLoggingAspect.java:74)
Actually, there were zero interactions with this mock.

[ERROR] Errors: 
[ERROR]   TestLoggingAspect.testLogSuccess:49 NullPointer Cannot invoke "com.skillstorm.taxservice.aspects.LoggingAspect.log(org.aspectj.lang.ProceedingJoinPoint)" because "this.loggingAspect" is null
[INFO] 
[ERROR] Tests run: 281, Failures: 1, Errors: 1, Skipped: 0
``` 

Was thinking it was because other PRs are waiting for changes to get merged, but I've approved / merged many of them now... we'll see once they all get merged

**Review by brittshook — CHANGES_REQUESTED** [review]
I'm still getting the TestLoggingAspect.testLogSuccess:49 NullPointer error. Can you check if you're getting this?

**Pull request merged**
